kube/src/api/object.rs

```diff
@@ -222,3 +222,22 @@ impl<'a, T: Clone> IntoIterator for &'a mut ObjectList<T> {
         self.items.iter_mut()
     }
 }
+
+impl<T: Clone> k8s_openapi::Resource for ObjectList<T> {
+    const API_VERSION: &'static str = "v1";
+    const GROUP: &'static str = "";
+    const KIND: &'static str = "List";
+    const VERSION: &'static str = "v1";
+}
```

Comment:
Comparing to k8s-openapi: https://docs.rs/k8s-openapi/0.10.0/src/k8s_openapi/v1_19/list.rs.html#5-18
This appears to be a dummy implementation. The correct should be `kind: NormalKindList` and `group: original-group`

Reply:
Indeed, ObjectList should not implement Resource. My original intention is to implement Metadata and call the namespace and resourceVersion methods in general, but Metadata must implement Resource. Please help close this PR.